source/Halibut.Tests/PollingClientConnectionHandlingFixture.cs

```diff
@@ -101,6 +104,7 @@ public async Task PollingClientShouldReConnectQuickly(ClientAndServiceTestCase c
                 .WithPortForwarding()
                 .WithDoSomeActionService(doSomeActionServiceAction)
                 .WithEchoService()
+                .WithPollingReconnectRetryPolicy(() => new RetryPolicy(99999999, TimeSpan.Zero, TimeSpan.FromMinutes(1)))
```

Comment:
If we don't mark the last attempt as a success the time will be greater than zero resulting in a 1 min timeout.